packages/library/src/commonMain/kotlin/io/realm/PrimaryKey.kt

```diff
@@ -19,4 +19,13 @@ package io.realm
 @Retention(AnnotationRetention.SOURCE)
 @Target(AnnotationTarget.FIELD)
 @MustBeDocumented
+/**
+ * Annotation marking a field as a primary key inside Realm.
+ *
+ * Only one field in a RealmObject class can have this annotation, and the field should uniquely
+ * identify the object.
+ *
+ * It is allowed to apply this annotation on the following primitive types: String, Byte, Char,
```

Comment:
```suggestion
 * This annotation applies to the following primitive types: String, Byte, Char,
```
Passive -> active reword.